x-pack/plugins/security_solution/public/management/pages/host_isolation_exceptions/view/hooks.ts

```diff
@@ -66,7 +66,7 @@ export function useCanSeeHostIsolationExceptionsMenu(): boolean {
   useEffect(() => {
     async function checkIfHasExceptions() {
       try {
-        const summary = await getHostIsolationExceptionSummary(http);
+        const summary = await getHostIsolationExceptionSummary({ http });
```

Comment:
Adding the filter value as a second parameter will prevent you from doing this

Reply:
39e4ba34b1ad8e4f090551ba1f4c9aef6f5b1c83